examples/tokenvm/DEVNETS.md

```diff
@@ -92,7 +92,10 @@ Now we can spin up a new network of 6 nodes with some defaults:
 --avalanchego-release-tag v1.10.3 \
 --create-dev-machine \
 --keys-to-generate 5 \
---profile-name <AWS_PROFILE_NAME>
+--profile-name <AWS_PROFILE_NAME> \
+--vm-binary-file <PATH_TO_VM_BINARY> \
+--chain-name <NAME> \
+--chain-genesis-file <PATH_TO_CHAIN_GENESIS_FILE>
```

Comment:
How do I provide the chain config and AvalancheGo config? Is it the same as the previous create subnet command?

Reply:
Use this to update flags: https://github.com/ava-labs/avalanche-ops/blob/main/avalancheup-aws/src/default_spec/mod.rs#L457-L501